examples/face_detector/ios/FaceDetector/ViewContoller/CameraViewController.swift

```diff
@@ -0,0 +1,274 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import AVFoundation
+import MediaPipeTasksVision
+import UIKit
+
+/**
+ * The view controller is responsible for performing detection on incoming frames from the live camera and presenting the frames with the
+ * bounding boxes of the detected objects to the user.
```

Comment:
Update the comments here: "bounding boxes ..." -> "bounding boxes and landmark of the detected faces to the user."